test/test_turbomole.py

```diff
@@ -0,0 +1,231 @@
+        """test for getting coord function"""
+
+        coord = np.array([    
+    0.00000000000000,      0.00000000000000,      1.24876020687021, 
+    0.00000000000000,      0.00000000000000,     -1.24876020687021,      
+    1.74402803906479,      0.00000000000000,      2.32753092373037,      
+   -1.74402803906479,      0.00000000000000,      2.32753092373037,      
+    1.74402803906479,      0.00000000000000,     -2.32753092373037,      
+   -1.74402803906479,      0.00000000000000,     -2.32753092373037,      
+    ])
+
+        coord_from_control = coord
+
+
+        np.testing.assert_almost_equal(coord, coord_from_control ,decimal = 10)
+
+
+
+
+    def test_density_matrix(self):
+        dm_t_1 = np.array([ [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], 
+                            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], 
+                            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], 
+                            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0] ])
+        
+        dm_t_2 = np.array(
+                        [ [1.750696427558272e-10, 0.0, 1.9635196878882362e-08, 1.0186172011065542e-07, 
+                          -3.2512605768905584e-08, -1.1467681121179046e-06, -1.742901089655016e-06, 1.3065414000075634e-05], 
+                        [1.9635196878882362e-08, -1.0186172011065542e-07, 6.146897206720771e-05, 0.0, -0.0006708768124209091, 
+                        -0.00010970050206673946, 0.007406443044944121, 0.0024794525900529852], 
+                        [-3.2512605768905584e-08, 1.1467681121179046e-06, -0.0006708768124209091, 0.00010970050206673946, 
+                          0.007517774936800343, 0.0, -0.08525940680476202, -0.013843028457903528], 
+                        [-1.742901089655016e-06, -1.3065414000075634e-05, 0.007406443044944121, -0.0024794525900529852, 
+                          -0.08525940680476202, 0.013843028457903528, 0.9924207559160625, 0.0]])
+
+
+
+        dm_t_3 = np.array(
+                        [[4.755043490192427e-10, 1.454028420503369e-26, 7.649642005146595e-08, 3.5835856618061715e-07, 
+                        -1.3427109547686181e-07, -3.08127223073118e-06, -6.891696394388752e-06, 2.045390744453282e-05], 
+                         [7.649642005146596e-08, -3.583585661806172e-07, 0.0002823792558630928, -2.710505431213761e-20, 
+                         -0.0023437673262842137, -0.00039450553499091015, 0.014306163248332794, 0.008484358013417057], 
+                        [-1.3427109547686176e-07, 3.08127223073118e-06, -0.002343767326284214, 0.00039450553499091026, 
+                          0.02000458525053515, -8.673617379884035e-19, -0.13059544360067088, -0.05043394730559981], 
+                        [-6.891696394388754e-06, -2.0453907444532825e-05, 0.01430616324833279, -0.008484358013417057, 
+                         -0.13059544360067088, 0.050433947305599816, 0.9797130350180959, -1.3877787807814457e-17]])
+
+
+        with open("traj-0-log_0.yaml", "r") as f:
```

Comment:
This is reading data that was computed previously by a separate test? I'm not sure you can guarantee the order that tests run in. It would be better to make this part of the same test that generated the data.

Reply:
fixed it now (?)